### DISCUSSION TITLE
Search field in Docs does not work for versions below 4.0

### DISCUSSION THREAD

**Discussion by peterchrjoergensen** [laravel/nova-issues#5788] [General]
Tested on Windows 10 via Chrome.

https://nova.laravel.com/docs/3.0/installation.html
![image](https://github.com/laravel/nova-issues/assets/44966738/08fd8652-8517-4b5f-9057-7313f8a1ca41)

https://nova.laravel.com/docs/4.0/installation.html
![image](https://github.com/laravel/nova-issues/assets/44966738/48708712-dbe3-48c4-945c-705774fb868a)

**Comment by crynobone** [comment]
I believe you're still browsing the old documentation, Nova 3 documentation is no longer hosted on the site and now only accessible from GitHub: https://github.com/laravel/nova-docs/tree/3.x

![CleanShot 2023-08-09 at 16 28 16](https://github.com/laravel/nova-issues/assets/172966/4586c189-459a-40fe-bafd-d89288cc2fd1)

**Reply by peterchrjoergensen** [reply]
You're correct, thanks! 

I'm going by nova.laravel.com and then clicking on "Documentation" in the navbar.

If I do this in Chrome, my daily driver, I see the old documentation.

If I do this in Firefox, I see the new documentation.

Clearing the cache in Chrome (Ctrl+Shift+R) solved it and leads me to the new documentation.
